pkg/collector/corechecks/snmp/internal/checkconfig/config_metadata.go

```diff
@@ -86,6 +86,74 @@ var LegacyMetadataConfig = MetadataConfig{
 	},
 }
 
+var TopologyMetadataConfig = MetadataConfig{
```

Comment:
Yes, indeed, I though about that. 

Right now, there is no much benefit of using a profile for topology since LLDP metadata definition doesn't change for different vendors/devices. It's similar for Routing Table and ARP Cache data that we might want to include in the future.

There is the case of CDP (http://www.mibdepot.com/cgi-bin/getmib3.cgi?win=mib_a&r=cisco&f=CISCO-CDP-MIB-V1SMI.my&v=v1&t=tree), Cisco version of LLDP, but the data structure and fields are different so difficult to fit in a single schema that we can reuse for both LLDP and CDP.